docs/google-kubernetes-tutorial.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Deploy TiDB to Kubernetes on Google Cloud
+summary: Tutorial for deploying TiDB on Google Cloud using Kubernetes.
+category: operations
+---
+
+# Deploy TiDB, a distributed MySQL compatible database, to Kubernetes on Google Cloud
```

Comment:
How about keeping this title consistent with the one in the metadata?

Reply:
Done.